data/transportation/street-network-analysis/index.html

```diff
@@ -16,6 +16,12 @@
       label: Street Network
   updates:
     - May 20, 2020
+MultimodalNetwork:
+  downloads:
+    - url: https://drive.google.com/uc?id=1DK1eYvl7NAHeL0zdFe8MgwSP2NA9XQcN&export=download
+    label: Multimodal Network
```

Comment:
issue: your spacing is off here.
```suggestion
      label: Multimodal Network
```